src/Evolution/DiscontinuousGalerkin/MortarTags.hpp

```diff
@@ -29,6 +30,21 @@ struct MortarData : db::SimpleTag {
       std::unordered_map<Key, evolution::dg::MortarData<Dim>, boost::hash<Key>>;
 };
 
+/// History of the data on mortars, indexed by (Direction, ElementId) pairs, and
+/// used by the linear multistep local time stepping code.
+///
+/// The `Dim` is the volume dimension, not the face dimension.
+template <size_t Dim, typename CouplingResult>
```

Comment:
The naming originated with me, so I'll comment here.  `CouplingResult` is the result of calling a functor of type `Coupling` used in `BoundaryHistory`.  I don't think it would be accurate to call it a "mortar history item", because it is not the type stored in the history (except internally as a cache).  It is also the result of `LtsTimeStepper::compute_boundary_delta`, which again has a `Coupling` template parameter.  If it is going to be renamed, it should be done consistently, which I think is out-of-scope for this PR.

A comment pointing to one of those two places might make sense, though.

Reply:
Thanks for the clarification @wthrowe. I agree with keeping the name, and that pointing to one of the LTS places where it comes from would help